backend/routes/api/parks.js

```diff
@@ -132,13 +133,26 @@ function buildDateRangeObject(dateRange, readyToPublish) {
 
 // build a current season object
 function buildCurrentSeasonOutput(seasons) {
```

Comment:
This is always called but the value is only conditionally used in the output. The call should be moved inside the condition.